### PR TITLE
article: candian-bitcoin-owners-statistics-2021.md

### PR DESCRIPTION
**Pull request by barborawolf**
to do:
Grammarly check
picture 
formatting check

### CODE REVIEW

File: content/articles/candian-bitcoin-owners-statistics-2021.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+title: Bitcoin ownership among Canadians in 2021
```
Comment (nourspace) [inline content/articles/candian-bitcoin-owners-statistics-2021.md]: Capitalize Titles

File: content/articles/candian-bitcoin-owners-statistics-2021.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+title: Bitcoin ownership among Canadians in 2021 
+description: According to the Bank of Canada’s Financial System Review, cryptoassets market have seen rapid growth and integration into the traditional finance in recent years
+img: https://user-images.githubusercontent.com/59807305/195985948-607de861-6ce7-41f5-813e-a777403a11bd.jpg
```
Comment (nourspace) [inline content/articles/candian-bitcoin-owners-statistics-2021.md]: here we put the image URL only without markdown syntax.

File: content/articles/candian-bitcoin-owners-statistics-2021.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+### About 90% of Canadians are aware of Bitcoin
+Yet the survey in 2021 found that 40% of Bitcoin owners poses a low level understanding of how the asset works. This was also true for 66% of those who did not own Bitcoin.
+
+### Purchases were largely driven by investment reasons 
+Half of all owners in 2021 purchased Bitcoin in 2020 or 2021. "These recent owners are even more likely than earlier adopters to view Bitcoin as an investment vehicle." - Bank of Canada
+https://user-images.githubusercontent.com/59807305/195988874-07ce9982-4ae4-4c11-8794-cc8f9c8bdef9.jpg
+
+### Most canadian Bitcoin owners hold small amounts
+"In 2021, the median amount of Bitcoin held was Can$500 worth, and 70% of Bitcoin owners held the equivalent of Can$5,000 or less." Bank of Canada
```
Comment (nourspace) [inline content/articles/candian-bitcoin-owners-statistics-2021.md]: this is a quote so better using

```
> something something
Bank of Canada
```

File: content/articles/candian-bitcoin-owners-statistics-2021.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+
+> The Bank of Canada has been studying Bitcoin since 2016 using surveys. The 2021 Bitcoin Omnibus Survey provides primarily information about awareness, ownership and usage of Bitcoins among Candadians. 
+
+## Five Takeawyas from the 2021 Bitcoin Omnibus Survey
+
+### Sharp increase of Bitcoin ownership in 2021
+Ownership of Bitcoin in 2021 has seen a steep rise in 2021 after keeping under 5% from 2018 to 2020. The report also uncovered that "Purchasers used mobile apps much more than web-based exchanges, mining or visiting a Bitcoin ATM." - Bank of Canada
+
+https://user-images.githubusercontent.com/59807305/195987949-aa796718-4f8c-4019-bca6-8d67a9ea3766.jpg
```
Comment (nourspace) [inline content/articles/candian-bitcoin-owners-statistics-2021.md]: what is this image url doing here?

**Review by nourspace — CHANGES_REQUESTED** [review]

**Comment by nourspace** [comment]
In general, it would be nice to have some images from the article. Also, adding sources at the end is super important.

File: content/articles/candian-bitcoin-owners-statistics-2021.md (hunk 4 above)
Comment (barborawolf) [reply content/articles/candian-bitcoin-owners-statistics-2021.md]: it's a graph. I saved it and uploaded it.

File: content/articles/candian-bitcoin-owners-statistics-2021.md (hunk 3 above)
Comment (barborawolf) [reply content/articles/candian-bitcoin-owners-statistics-2021.md]: It will looks weird. check

**Review by nourspace — APPROVED** [review]
Looks great

**Pull request merged**